src/account-settings/demographics/data/service.js

```
@@ -120,3 +120,22 @@ export async function patchDemographics(userId, commitValues) {
 
   return convertData(data, FROM);
 }
+
+/**
+ * retrieve the options for each field from the Demographics API
+ */
+export async function getDemographicsOptions() {
+  const requestUrl = `${getConfig().DEMOGRAPHICS_BASE_URL}/demographics/api/v1/demographics/`;
+  let data = {};
+
+  try {
+    ({ data } = await getAuthenticatedHttpClient().options(requestUrl));
+  } catch (error) {
+    // We are catching and suppressing errors here on purpose. If an error occurs during the
```

Just wanted to call attention to this comment. The decisions to hide or display fields in the DemographicsSection will be based on whether or not this data was successfully retrieved from this function call.

I'd rather catch and re-throw an error here that can be handled downstream but the changes to support this during Fetch/Get settings calls was non-trivial.

At the very least I wanted to document why we were catching and swallowing errors here.